### ISSUE TITLE
[feature request] Add support for multiple domains/all subdomains

### ISSUE THREAD

**Issue by sscholl** [dprandzioch/docker-ddns#11]
Hi,

thanks for this project. I'm using it for half a year now, it works fine. 
However, I use FritzBox as DDNS requester and it does not allow to send multiple requests. Can you add functionality, so that I can send mutliple domains (maybe separated by `,`)?

My current workaround is the following:
```
        result := UpdateRecord(response.Domain, response.Address, response.AddrType)
+       UpdateRecord("*." + response.Domain, response.Address, response.AddrType)
```

Thanks
sscholl

**Comment by dstapp** [comment]
Hey. Thanks! It's good to hear that this project helps you. 
Just to get that right, you want to do something like this:
`/update?secret=changeme&domain=foo,bar,baz&addr=1.2.3.4`

and have then `foo`, `bar` and `baz` all pointing to `1.2.3.4`?

**Comment by sscholl** [comment]
Hi David,
Thanks for your response. Yes this is what i need. In my case i would point
foo and *.foo to 1.2.3.4.
Best regards
Simon

**Comment by dstapp** [comment]
Sounds like a good idea to me, especially when you want to provide multiple vhosts e.g. on a web server.
In fact, I also had that problem before but I decided to go with subfolders instead of vhosts.

Since this will not break any existing functionality and should not require any change in usage by current users,
I think I'll add this. Just not sure when I'll able to make it.

Anyway, I'll ping you in this bug report when changes are ready.

1. Januar 2018 00:45, "Simon Eric Scholl" <notifications@github.com (mailto:%22Simon%20Eric%20Scholl%22%20<notifications@github.com>)> schrieb:
Hi David,
Thanks for your response. Yes this is what i need. In my case i would point
foo and *.foo to 1.2.3.4.
Best regards
Simon

**Comment by dstapp** [comment]
@sscholl Can you please have a look at the latest `davd/docker-ddns:develop` image and let me know if you can verify the functionality works as expected? You can now provide a list of domains separated by comma.

**Comment by sscholl** [comment]
@dprandzioch Hey. I looked over the commit. I looks good so far, but I am not a Go-Dev. Does that mean, that the domain param needs to end with an "," ?

Thanks!
Simon

**Comment by dstapp** [comment]
@sscholl No, it may not end with an ",". You probably saw the unit test verifying this :)
Format has to either be "?domain=foo" or "?domain=foo,bar,baz".

**Comment by sscholl** [comment]
@dprandzioch looks great then!

**Comment by dstapp** [comment]
@sscholl This feature is now released with 1.2.0 and available in :latest images. Hope this helps. If there's any further problems, feel free to open a new bug report.

**Comment by sscholl** [comment]
works like a charm. Thanks a lot!!!
